flink-connectors/flink-jdbc/src/main/java/org/apache/flink/api/java/io/jdbc/JDBCOutputFormat.java

```diff
@@ -50,6 +50,8 @@
 	final JdbcInsertOptions insertOptions;
 	private final JdbcExecutionOptions batchOptions;
 
+	private static final int connectionCheckTimeout = 60;
```

Comment:
I think the variable name should be  `CONNECTION_CHECK_TIMEOUT`, because it's static final variable.